pkg/ebpf/btf.go

```diff
@@ -191,8 +225,11 @@ func (b *orderedBTFLoader) checkForMinimizedBTF(extractDir string) (*returnBTF,
 		if err != nil {
 			return nil, err
 		}
-		// no module load function for minimized single file BTF
-		return &returnBTF{vmlinux: spec, moduleLoadFunc: nil}, nil
+		b.resultMetadata.filepathUsed = btfRelativePath
```

Comment:
Below you use the absolute path, but here you use the relative path. I think we should probably use the absolute path everywhere.

Reply:
Changed to absolute path everywhere